package.json

```diff
@@ -4,7 +4,7 @@
   "description": "Physics system for A-Frame VR, built on Cannon.js & Ammo.js",
   "main": "index.js",
   "scripts": {
-    "dev": "budo index.js:dist/aframe-physics-system.js -d examples --port 8000 -- -g browserify-shim",
+    "dev": "budo index.js:dist/aframe-physics-system.js -d examples --port 8000 --ssl -- -g browserify-shim",
```

Comment:
Please note that generating the self-signed certificate doesn't work on Ubuntu 22.04, that's one of the reason we replaced budo by webpack dev server in aframe repo. Also see #22

Reply:
Ah, I wasn't aware of that. :-(